daemon/src/setup_maker.rs

```diff
@@ -69,8 +67,7 @@ impl Actor {
         ),
     ) -> Self {
         Self {
-            db,
-            process_manager,
+            executor: command::Executor::new(db, process_manager),
```

Comment:
perhaps we could clone the executor and pass it instead of the `db` and `process_manager` to encapsulate what can be done with them?

Reply:
Yes. I just wanted to keep the noise down in the PR and not ripple the changes through too much!